tools/run_tests/xds_k8s_test_driver/tests/change_backend_service_test.py

```diff
@@ -71,9 +71,8 @@ def test_change_backend_service(self) -> None:
             self.td.create_forwarding_rule(self.server_xds_port)
 
         with self.subTest('05_start_test_servers'):
-            self.default_test_servers: List[
-                _XdsTestServer] = self.startTestServers()
-            self.same_zone_test_servers: List[
+            default_test_servers: List[_XdsTestServer] = self.startTestServers()
+            same_zone_test_servers: List[
```

Comment:
I really like how you reformatted it in tools/run_tests/xds_k8s_test_driver/tests/failover_test.py:73 - split type declaration and the assignment.

Reply:
Updated all test files under `tests/*.py`. This makes the code more readable that at a glance we know certain fields are living at the scope of the method.